### PR TITLE
ECDC-2340: Missing method in EpicsKafkaForwarderControl

### PR DESCRIPTION
**Pull request by ebadkamil**
This PR 

- Deal with status message stream from the forwarder which does not contain `converters` key.  
- Add `pv_forwarding_info` method in `EpicsKafkaForwarderControl`

### CODE REVIEW

File: nicos_ess/devices/forwarder.py [hunk 1]
```diff
@@ -106,18 +106,21 @@ def status_update(self, message):
...
             pvs_read = []
             not_forwarded = []
             for stream in msg["streams"]:
                 pv = stream["channel_name"]
                 pvs_read.append(pv)
                 if pv in issued:
-                    forwarding = is_forwarding(issued[pv][0], issued[pv][1],
-                        stream["converters"])
+                    try:
+                         # 'converters' key doesn't exist in stream. Legacy?
```
Comment (mattclarke) [inline nicos_ess/devices/forwarder.py:116]: If it doesn't exist then delete the code!

**Comment by mattclarke** [comment]
This should go into Gerrit when it is ready

File: nicos_ess/devices/forwarder.py (hunk 1 above)
Comment (ebadkamil) [reply nicos_ess/devices/forwarder.py:116]: What about  `add_legacy` method on line 132? I am not sure if latest version of forwarder handle such commands?

Comment (mattclarke) [reply nicos_ess/devices/forwarder.py:116]: delete delete delete!
:)

Comment (ebadkamil) [reply nicos_ess/devices/forwarder.py:116]: that is fun :D

Comment (mattclarke) [reply nicos_ess/devices/forwarder.py]: Somebody said: if you get paid per line of code then you should get paid double for each line you delete!

Comment (ebadkamil) [reply nicos_ess/devices/forwarder.py]: Have to update the tests now.

**Comment by ebadkamil** [comment]
Tests are now updated.

**Comment by mattclarke** [comment]
Is there a Gerrit PR we can link to?

**Comment by ebadkamil** [comment]
Not yet. Will do it today most likely if you think it's good to go there.

**Comment by mattclarke** [comment]
Gone into Gerrit

**Pull request closed without merging**